docs/crio.conf.5.md

```diff
@@ -81,7 +81,7 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
   If true, the runtime will not use `pivot_root`, but instead use `MS_MOVE`.
 
 **conmon**="/usr/local/libexec/crio/conmon"
-  Path to the conmon binary, used for monitoring the OCI runtime.
+  Path to the conmon binary, used for monitoring the OCI runtime. Will be searched inside $PATH if empty.
```

Comment:
if you take suggestion above, use here too please.

Reply:
Yes, I changed it as suggested. :+1: